### PR TITLE
DS-7260: First Integration of Postman with torpedo

### PR DESCRIPTION
**Pull request by jkagliwal-px**
This PR is the first step to integrate Postman-Newman feature for Api-Load-Testing into Torpedo framework.

This is very bare minimum and have some static mechanism, like replacing of Collection files manually, which will be automated in later PRs.

The Code does the following -
1.Accepts the Collection Json which we will placed manually by the user in /drivers/postmanApiLoadDriver/collections folder
2. The newman command will be executed against the provided json file.

Ticket resolved from this PR is - https://portworx.atlassian.net/browse/DS-7267

Jenkins Run Logs - 
https://jenkins.pwx.dev.purestorage.com/job/PDS/job/pds-system-test/job/pds-bkp-restr-system-test/116/console

### CODE REVIEW

File: drivers/postmanApiLoadDriver/DockerFile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM centos:centos7
```
Comment (mkoppal-px) [inline drivers/postmanApiLoadDriver/DockerFile:1]: Why do we need a separate DockerFile for this? Can we not run the `newman run` command in the torpedo pod itself?
I think we just need to add the newman utility to existing torpedo image.

Comment (jkagliwal-px) [reply drivers/postmanApiLoadDriver/DockerFile:1]: As suggested, I have added the newman dependencies in torpedo DockerFile and have wriiten a new function to just run the newman command in torpedo pod.
Howerver I would suggest not to remove "run newman as a separate pod" mechanism for future use if need be.

File: drivers/postmanApiLoadDriver/postmanApi.go [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+	if strings.Contains(output, "failure") {
+		log.FailOnError(err, "newman exited with a failure.. Please check logs for more details")
+	}
+	k8sCore.DeletePod(postDep.Name, postDep.Namespace, true)
+	return postDep, true, nil
+}
+
+func ExecutePostmanCommandInTorpedo() (bool, error) {
+	collectionPath, err := GetPostmanCollectionPath()
+	if err != nil {
```
Comment (dbhatnagar-px) [inline drivers/postmanApiLoadDriver/postmanApi.go:163]: Same comment - FailOnError checks internally - but you dont need it here. You should return err and handle it in the testcase

File: tests/pds/pds_postman_test.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+			ctx, err := GetSourceClusterConfigPath()
+			log.FailOnError(err, "failed while getting dest cluster path")
+			postmanParams := postmanLib.PostmanDriver{
+				ResultsFileName: resultsFileName,
+				ResultType:      "json, cli",
+				Namespace:       params.InfraToTest.Namespace,
+				Replicas:        2,
+				Iteration:       "2",
+				Kubeconfig:      ctx}
+			postmanLib.GetProjectNameToExecutePostman("pds", &postmanParams)
```
Comment (dbhatnagar-px) [inline tests/pds/pds_postman_test.go:26]: Please save the output in an output file as well.

**Review by dbhatnagar-px — CHANGES_REQUESTED** [review]

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by dbhatnagar-px — APPROVED** [review]
lgtm

**Pull request merged**